iocage/cli/stop.py

```diff
@@ -34,8 +34,11 @@
 @click.option("--rc", default=False, is_flag=True,
               help="Will stop all jails with boot=on, in the specified"
                    " order with higher value for priority stopping first.")
+@click.option("-f", "--force", default=False, is_flag=True,
+              help="Skip all pre-stop actions like stop services and"
+                   " gently shutdown, and kill the jail process.")
```

Comment:
This sentence flow can be confusing. Suggestion: break into two sentences: "Skips all pre-stop actions like stop services. Gently shuts down and kills the jail process."

Reply:
Thank you @Mrt134 for suggestion. I changed help text.

@skarekrow I think that force stop should skip only pre-stop actions, but it should execute post-stop, because user could define own post-stop actions.
The reason for adding `--force` was the inability to run specific pre-stop actions :)